server/src/main/java/com/coincoinche/websockets/controllers/GameController.java

```diff
@@ -33,9 +67,42 @@ public GameController() {
   public void getNewHandForRound(
       @DestinationVariable String gameId, @DestinationVariable String username) {
     // TODO error handling if the game is not found
+    CoincheGame game = this.store.getGame(gameId);
+
+    this.template.convertAndSend(
+        getTopicPath(gameId, username), new RoundStartedEvent(game.getPlayer(username).getCards()));
+
     this.template.convertAndSend(
-        String.format("/topic/game/%s/player/%s", gameId, username),
-        new RoundStartedEvent(this.store.getGame(gameId).getPlayer(username).getCards()));
+        getTopicPath(gameId, username),
+        new RoundPhaseStartedEvent(RoundPhase.BIDDING, game.getCurrentPlayerIndex()));
+
+    this.notifyPlayerTurnStarted(gameId, username);
+  }
+
+  /**
+   * To be called by the client when the player bids.
+   *
+   * @param gameId - id of the game
+   * @param username - username of the player
+   */
+  @MessageMapping("/game/{gameId}/player/{username}/bid")
+  public void makeBid(
+      @DestinationVariable String gameId,
+      @DestinationVariable String username,
+      @Payload PlayerBadeEvent event) {
+    CoincheGame game = this.store.getGame(gameId);
+    BiddingMove move = BiddingMove.fromEvent(event);
+    try {
+      move.applyOnGame(game);
+      this.template.convertAndSend(getBroadcastTopicPath(gameId), event);
+    } catch (IllegalMoveException e) {
```

Comment:
I guess in this case for bidding we want the player to pass. For playing, we would want the player to play a random card, right?

Reply:
Good point, I implemented that 👍